### PR TITLE
[KER-2024]: update organizers, logo, sponsors..etc

### PR DESCRIPTION
**Pull request by humblec**
Added more details about the event.

### CODE REVIEW

File: content/events/2024-kerala/welcome.md [hunk 1]
```diff
@@ -1,87 +1,44 @@
...
+          <a class="btn btn-secondary btn-block" href="/events/2024-kerala/contact"> <i class="fa fa-envelope-o fa-lg"></i>&nbsp;&nbsp;
+            &nbsp;Contact the Organizers</a>
+        </div>
+      </div>
+    </div>
+  </div>
+
+  <div class="col-md-7">
+    <h3>Our program is now live!</h3>
+    <p>We are incredibly proud to be able to announce our program for our 2024 event which you can find <a href="/events/2024-kerala/program">here</a>. With a breadth of speakers from many backgrounds we are sure that their talks will entertain and engage you into the Open Spaces in the afternoons. If you don't already have a ticket then you can sign up to our event <a href="https://konfhub.com/devopsdays-kerala-2024">here</a>. We cannot wait to see you in September.</p>
```
Comment (phrawzty) [inline content/events/2024-kerala/welcome.md:36]: Your programme has not, as far as I can tell, been announced. The `program` page is empty. Are you should about this wording?

Also, once you _are_ ready to announce, you can (and should) use built-ins for this. See https://github.com/devopsdays/devopsdays-web/pull/14382/files#diff-81f19987eef556de82f5927b0a05a7b9e0ca6115b4c61261adc2b3881a34fb09L71 (which you are deleting in this PR) for an example.

File: content/events/2024-kerala/welcome.md [hunk 2]
```diff
@@ -1,87 +1,44 @@
...
+<div class="row">
+  <div class="col-md-4">
+    <img alt="DevOpsDays kerala 2024" src="/events/2024-kerala/logo.jpeg" class="img-fluid">
+    <div class="d-flex flex-row">
+      <div class="col-md-12">
+        <div class="p-2">
+          <a class="btn btn-secondary btn-block" href="/events/2024-kerala/sponsor"> <i class="fa fa-money fa-lg"></i>&nbsp;&nbsp;&nbsp;Sponsor the Conference</a>
+        </div>
+        <div class="p-2">
+          <a class="btn btn-secondary btn-block" href="/events/2024-kerala/program"> <i class="fa fa-book fa-lg"></i>&nbsp;&nbsp;
```
Comment (phrawzty) [inline content/events/2024-kerala/welcome.md:20]: Your programme has not, as far as I can tell, been announced. The `program` page is empty. Are you should about this button?

File: data/events/2024/kerala/main.yml [hunk 3]
```diff
@@ -51,17 +52,27 @@ nav_elements: # List of pages you want to show up in the navigation of your page
 # These are the same people you have on the mailing list and Slack channel.
 team_members: # Name is the only required field for team members.
   - name: "Dipu Devarajan"
+    linkedin: "https://www.linkedin.com/in/dipu-devarajan-11284b46/"
+    image: "/events/2024-kerala/organizers/dipu.jpeg"
```
Comment (phrawzty) [inline data/events/2024/kerala/main.yml:56]: We don't use paths here. Please see the [documentation](https://github.com/devopsdays/devopsdays-web/blob/main/themes/devopsdays-theme/reference.md#organizer-fields) for details, and literally any other `main.yml` that has organiser photos for examples.

**Review by phrawzty — CHANGES_REQUESTED** [review]

File: data/events/2024/kerala/main.yml (hunk 3 above)
Comment (humblec) [reply data/events/2024/kerala/main.yml]: Sure, removed Program for now..

File: data/events/2024/kerala/main.yml [hunk 4]
```diff
@@ -18,13 +18,13 @@ cfp_date_start:  # start accepting talk proposals.
 cfp_date_end:  # close your call for proposals.
```
[the remark this answers is not in the document]
Comment (humblec) [reply data/events/2024/kerala/main.yml]: Sure.. thanks ! addressed the same.

File: data/events/2024/kerala/main.yml [hunk 5]
```diff
@@ -18,13 +18,13 @@ cfp_date_start:  # start accepting talk proposals.
...
 cfp_date_announce:  # inform proposers of status
 
-cfp_link: "" #if you have a custom link for submitting proposals, add it here. This will control the Propose menu item as well as the "Propose" button.
+cfp_link: "https://talks.devopsdays.org/devopsdays-kerala-2024/cfp" #if you have a custom link for submitting proposals, add it here. This will control the Propose menu item as well as the "Propose" button.
 
 registration_date_start: # start accepting registration. Leave blank if registration is not open yet. This will make the "Register" button appear on your "Welcome" page.
 registration_date_end: # close registration. Leave blank if registration is not open yet. If you set "registration_date_start" you need a value here.
```
[the remark this answers is not in the document]
Comment (humblec) [reply data/events/2024/kerala/main.yml]: Done.

File: data/events/2024/kerala/main.yml [hunk 6]
```diff
@@ -40,6 +40,7 @@ nav_elements: # List of pages you want to show up in the navigation of your page
  # - name: registration
```
[the remark this answers is not in the document]
Comment (humblec) [reply data/events/2024/kerala/main.yml:40]: Awesome.. committed the changes as suggested

File: static/events/2024-kerala/devopsdays-kerala-2024-sponsorship-brochure.pdf
[the remark this answers is not in the document]
Comment (humblec) [reply static/events/2024-kerala/devopsdays-kerala-2024-sponsorship-brochure.pdf:1]: Sure .. added it in assets repo and here is the PR https://github.com/devopsdays/devopsdays-assets/pull/212 @nimbinatus .. Thanks a lot!

File: data/events/2024/kerala/main.yml [hunk 7]
```diff
@@ -37,9 +37,11 @@ location_address: "Le Méridien Kochi, Nettoor, Maradu, Kochi, Ernakulam, Kerala
 nav_elements: # List of pages you want to show up in the navigation of your page.
  # - name: propose
  # - name: location
- # - name: registration
+   - name: register
+     url: https://konfhub.com/devopsdays-kerala-2024
```
Comment (nimbinatus) [inline data/events/2024/kerala/main.yml:41]: Whoops, I had an extra set of spaces. That's why the preview doesn't show anything. If you can accept these changes (or update and commit yourself), I can merge. (p.s., if you want to check this locally, are comfortable doing so, and you already have hugo installed, a `hugo serve` works just fine.)
```suggestion
  - name: register
    url: https://konfhub.com/devopsdays-kerala-2024
```

**Review by nimbinatus — CHANGES_REQUESTED** [review]

**Comment by libreminds** [comment]
Thanks @nimbinatus . I have accepted the suggestion !! 👍

File: data/events/2024/kerala/main.yml (hunk 7 above)
Comment (libreminds) [reply data/events/2024/kerala/main.yml]: Thanks.. I missed it too.. Have accomodated the changes

**Comment by libreminds** [comment]
@nimbinatus looks we are good at rendering https://deploy-preview-14382--devopsdays-web.netlify.app/events/2024-kerala/welcome/

**Review by phrawzty — APPROVED** [review]

**Comment by humblec** [comment]
Thanks a lot @phrawzty and @nimbinatus for quick review and approval 🎉

**Pull request merged**